nilearn/datasets/func.py

```diff
@@ -2235,10 +2249,11 @@ def fetch_bids_langloc_dataset(data_dir=None, verbose=1):
     main_path = os.path.join(data_dir, main_folder)
     file_list = [
         os.path.join(path, f)
-        for path, dirs, files in os.walk(main_path)
+        for path, _, files in os.walk(main_path)
         for f in files
     ]
-    return os.path.join(data_dir, main_folder), sorted(file_list)
+    description = get_dataset_descr("bids_langloc")
+    return os.path.join(data_dir, main_folder), sorted(file_list), description
```

Comment:
Because this fetcher returns a tuple and not a Bunch, this may break the API of people who use this fetcher in their tutorial.

I'd be tempted to say we should not go through a deprecation because
- not returning a description feels more like a bug that it is being fixed
- dataset fetchers are more used for our examples are rarely used in "production" for research

Reply:
Ideally I would prefer to have these fetchers also return a bunch rather than a tuple, but I think this should be part of a different PR.

Reply:
Agreed. It's also good for consistency with the other fetchers

Reply:
> Because this fetcher returns a tuple and not a Bunch, this may break the API of people who use this fetcher in their tutorial.
> 
> I'd be tempted to say we should not go through a deprecation because
> 
>     * not returning a description feels more like a bug that it is being fixed
> 
>     * dataset fetchers are more used for our examples are rarely used in "production" for research

Not sure about that one, but you're probably right. +0

Reply:
Will at least mention it in the changelog in any case.